src/main/java/org/fundacion/jala/converter/models/parameter/PythonParameter.java

```diff
@@ -5,13 +5,15 @@
  * ("Confidential Information"). You shall not disclose such Confidential
  * Information and shall use it only in accordance with the terms of the
  * license agreement you entered into with Fundacion Jala
- */
-/**
- * @author Paola Aguilar
+ *
+ * @author Paola Ximena Aguilar Quiñones
  */
 package org.fundacion.jala.converter.models.parameter;
 
-public class PythonParameter extends Parameter{
+/**
+ * This class is to python parameter.
```

Comment:
..is for Python parameter, could be a better description.

Reply:
ok